src/shared/commons/collections-helpers.ts

```diff
@@ -0,0 +1,14 @@
+export class CollectionsHelpers {
+  constructor() {
+  }
+
+  filterElement(el) {
```

Comment:
a bit misleading method it's not doing a filtering it's just doing a null check. Also, not operating on a collection. Perhaps better removing it

Reply:
i will  remove it.